spim_core/spim_base.py

```diff
@@ -112,11 +112,12 @@ def log_to_file(self, filepath: Path, logger: Logger = None,
             log_handler.close()
             logger.removeHandler(log_handler)
 
-    def run(self, overwrite: bool = False):
+    def run(self, overwrite: bool = False, spim_name='micr'):
```

Comment:
Let's derive `spim_name` from the config instead of passing it in as a parameter where we only take the default.
We could call it `instrument_name` or `instrument_class` or `instrument_type` and possibly put it under `design_specs`.

I'm happy to do this if you're busy.

Reply:
Agreed, I am trying to remind myself and I can't even find where this run() function is called in either exa-spim-control or exa-spim-ui. @micahwoodard do you know where this actually gets called? I agree with Sonya that we should generalize the name of the input to ```instrument_name``` and add this to the config.toml file.

@micahwoodard could you take a quick crack at this? i think it should be quick.